media/js/plugincheck/tmpl/plugincheck-redesign.ui.tmpl.js

```diff
@@ -0,0 +1,45 @@
+/* exported featuredPluginsTmpl, otherPluginsTmpl, unknownPluginsTmpl */
+
```

Comment:
Just out of curiosity, are these template strings generated by something (I see that `exported` comment above...)? Or do we code by hand?